src/cortxfs/cortxfs_fops.c

```diff
@@ -103,8 +104,8 @@ int cfs_creat_ex(struct cfs_fs *cfs_fs, cfs_cred_t *cred, cfs_ino_t *parent,
 
 	RC_WRAP(kvs_begin_transaction, kvstor, &index);
 
-	RC_WRAP_LABEL(rc, out, cfs_creat, cfs_fs, cred, parent, name,
-		      mode, &object);
+	RC_WRAP_LABEL(rc, out, cfs_creat, parent_fh, cred, name, mode,
+		      &object);
```

Comment:
Suggestion: Can we add one more goto label for cleanup  instead of out?

Reply:
Done addressed at 5e7dbb1977ba167c578c886c4a4f68cb69b25acd